src/Routes.js

```diff
@@ -53,6 +55,7 @@ export const Routes = () => {
   const edgeParityInventoryListEnabled = useFeatureFlag(
     'edgeParity.inventory-list'
   );
+  const edgeGroupsUse = useEdgeGroups(false);
```

Comment:
It seems the hook argument false is not needed as the hook does not accept [arguments](https://github.com/RedHatInsights/insights-inventory-frontend/pull/2086/files#diff-0f486407b67781cb20ec63cea192aea4cdb84b5bbf0dec3b84feb56ee6ba4f28R5)